### ISSUE TITLE
Readme for keeping in progress work up to date (re NT email)

### ISSUE THREAD

**Issue by jbolmarcich** [LIS462Omeka/KillgroveTheme#5]
Nikki- as I've added everyone to this 'org' and team, you can commit/push/pull freely. Work from the branch named 'nikki', and we can keep the original 'thanks roy' fork as our baseline. 

Feel free to make as many branches as you want/need - maybe work on CSS in one, PHP in another, etc. Whatever feels comfortable to everyone.

**Comment by nikki-t** [comment]
Thanks Jen! I will upload the appropriate files and change the code on the branch named "nikki." I am not entirely comfortable with the push/pull process yet so maybe we can look at this briefly this Saturday (3/19).

**Comment by jbolmarcich** [comment]
Basically, you clone a set of files (forking). (Snap 23, per Ethel's photo
slang) Now it's your copy ( snap 1). Then you edit and make changes.
Pushing the edits to your repos. is making a commit. (Snap 2)
If you want to offer your improved files back to the orig project, you send
the pull request to the project owner. Then they can pull some or all of
your work into the original. (Snap 24).

Make a bit more sense? Or have made it more confusing?

I've just updated the team repository LIS462Omeka nikki branch with your
updates.

jlb

**Comment by nikki-t** [comment]
Okay I think I got it!
My branch (nikki) is currently up-to-date (3/15) with all of the customizations I have done in the theme-thanksroy-NTtest directory on dany. I created separate branches for the plugin customizations I did in case we want to change them later.

**Comment by jbolmarcich** [comment]
I think you can actually just put the plugin styling into the theme files,
per this structure. Then the theme overides the plugin default just like it
overrides the omeka default.
https://omeka.org/codex/Theming_Plugin_Pages

I've forked the geolocation plugin b/c I'm trying to sub a different set of
map tiles that fit better with our site's feel. If I can't make that work,
there is some styling that can be requested as part of the Google maps API
call.

But for ex), we can theme the image viewer display to feel like the rest of
the site rather than being a black box.

See project map on LIS462 for thoughts.

jlb

**Comment by nikki-t** [comment]
Yes I just had to create a new folder in our test theme directory for each plugin and make sure that each directory structure matches that of the plugins in the main killgrovepapers directory. This way I am not messing with anything outside of the test theme directory. Does that make sense?

I can look through the image viewer to see if I can edit the style and will place the customized files in the test directory.

Where is the project map?

Thanks,
NT

**Comment by jbolmarcich** [comment]
It's a separate repo in LIS462Omeka.
https://github.com/LIS462Omeka/ProjectMap

**Comment by jbolmarcich** [comment]
Just wanted to say thanks for you all being so good with this coding. I
thought I knew a lot more than I really do :)

That said, I'm happy to learn whatever and please let me know whatever else
I can do to be useful in this process.

jen c

**Comment by jbolmarcich** [comment]
Jen et al.-

JC - Could you touch base with Lizzie and Michelle and get an idea of what
they're thinking in terms of adding content? We could do an exhibit, or
just context pages, or something else. Not sure what they're thinking in
terms of impt. material to add (and what should be in metadata vs article
type info...)

We need to plan out how all this going to appear beyond the wireframe.

Things needing doing:

We need a defined color palette, and buttons, tabs, graphics, icons styling
decisions and files found/made. And a final font(s) choice too.

Sitemap, including map browse page (do we want that, or just the maps on
the item pages?)

Timeline? See content note above.

Item view page - what metadata do we want showing and how do we want it
labeled?

Plugins:

Scripto - I'll need Travis to install Mediawiki, but then we can load the
Scripto stuff. That will need an interface too. Check out MHC's site (
transcribe.mtholyoke.edu) and SI, etc.

Universal Viewer - can be themed to fit site - see color/design choices.
Once those are made, we can tweak it.

Subject Browse - Nikki's working on this CSS, but it actually falls under
the design choices too.

Geolocation - I'm tweaking this to use these map
http://maps.stamen.com/watercolor/#5/9.025/38.747 tiles but Google's
labels. We can change the map marker too. See colors above.

ExhibitBuilder - makes set of pages for exhibit style interface (think
DPLA's). See content choices and any specific designs needing doing.

I've outlined this more broadly @ LIS462Omeka/ProjectMap
http://github.com/LIS462Omeka/ProjectMap

Frankly, I'm seeing us as a coop more than two teams at this point. :-)

That's everything in my head at the moment. :-)
jlb

**Comment by nikki-t** [comment]
I will try to write something up this week about design choices: font/colors/graphics etc. and then post it to Trello so everyone can edit and refine.

I have been emailing with Jana about a timeline and need to check in with Ann and Ellen but I am thinking that we are doing the usability test 3/26 and then running a live class evaluation of the interface on 4/9 and refine until delivery date of 4/16 or 4/23. Let me know if this works for you or if we need to make changes! At this point these are all flexible dates.

-NT

**Comment by nikki-t** [comment]
I cannot figure out how to change the background color of the UniversalViewer! Any suggestions?

**Comment by jbolmarcich** [comment]
what color do you want?
gimme the hex and I'll figure it out.

**Comment by nikki-t** [comment]
I think white is good! Just to counterbalance the black. #FFFFFF
Let me know how you do it!
I tried pulling the css file from the javascript directory and recreating the directory in the theme directory with no luck.

**Comment by nikki-t** [comment]
I uploaded some changes I made to the php to customize the browse items, search results and item page. See documentation for steps. I am waiting on doing anymore until we see what metadata elements are defined by Metadata group in their documentation.

One question, if we wanted to place the UniversalViewer above the metadata on the item page how do we change the order on the page of where the plug in shows up?

Thanks,
Nikki
